packages/pwa-kit-create-app/scripts/build.js

```diff
@@ -48,12 +50,12 @@ const main = () => {
 
     return Promise.all(
         pkgNames.map((pkgName) => {
+            const actualPkgName =
+                pkgName === EXTENSION_BASE_NAME ? EXTENSION_BASE_NAME : `${TEMPLATE_PREFIX}${pkgName}`
```

Comment:
What is the purpose of this logic? Is it because in a preset we have the pkgName as `extension-base`?